### DISCUSSION TITLE
what is NER?

### DISCUSSION THREAD

**Discussion by bmaluijb** [deepgram/community#214] [General help] [answered]
Hi!

Maybe a dumb question. What is NER? I see it when I inspect my Deepgram usage log, amongst the features used in a call, like Diarize = true. It says "NER" = true. What features does it stand for? I couldn't find it in any docs.

Thank you very much for the answer folks.

**Comment by jkroll-deepgram** [comment]
Thanks for the question! NER stands for Named Entity Recognition, i.e. being able to recognize many types of proper names, including of people, places, companies, and addresses. You can read more about Deepgram's NER in our [Entity Detection documentation](https://developers.deepgram.com/docs/detect-entities). You can choose to use or not use this option in our API with the parameter `detect_entities={true|false}`.

**Comment by bmaluijb** [comment]
Aha, thank you very much for the quick response.

I’m using the .NET SDK with the nova model (by indicating (tier = "nova"). I haven't set NER to true, nor does the URL in the usage log show it. (It looks like this: POST /v1/listen?tier=nova&punctuate=true&diarize=true&smart_format=true&utterances=true&paragraphs=true).

So why does the log say it's true? Maybe an error in the log? And am I paying for it?

Also, in the .NET SDK, I have two options for it: DetectEntities and NamedEntityRecognition in the PrerecordedTranscriptOptions class. Are they both for the same feature?

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by jkroll-deepgram** [comment, marked as answer]
I got additional clarity on this - NER is a holdover from an older feature, and you can disregard the references to it in the logs. You're using Nova with [Smart Formatting](https://developers.deepgram.com/docs/smart-format), which is the most accurate way to format entities, and we are continually making enhancements, most recently last week - see our [Changelog entry](https://deepgram.com/changelog/smart-formatting-improvements/). 

Entity Detection (DetectEntities in the .NET SDK) is another feature you can enable, if you want to receive an additional `"entities"` section within your API request response, giving you information on the label, value, confidence, start word, and end word of each detected entity.

Pricing starts by model, and using the Nova model, you can enable or disable Smart Formatting with no change to cost. The NER that appears in the logs also does not affect the cost. Entity Detection is an add-on that incurs an additional cost per minute. When logged into the Deepgram console, you can also go to Settings -> Billing & Plans to see the pricing breakdown of each model and add-ons.

**Comment by bmaluijb** [comment]
Thank you for the info!

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________
